goty-client/src/app/results/results-table/results-table.component.html

```diff
@@ -0,0 +1,26 @@
+<app-info [title]="title">
+  <div class="table">
+    <p-table
+      [autoLayout]="true"
+      [value]="values"
+      [paginator]="true"
+      [rows]="10"
+      [rowsPerPageOptions]="[10, 20, 50]">
+      <ng-template *ngIf="!stringArr" pTemplate="header">
+        <tr>
+          <th *ngFor="let prop of this.columnConfig">{{prop}}</th>
+        </tr>
+      </ng-template>
+      <ng-template pTemplate="body" let-row>
+        <tr [ngClass]="row.rank | resultsTableStylePipe: styleType">
+          <ng-container *ngIf="stringArr; else obj">
+            <td>{{row}}</td>
+          </ng-container>
+          <ng-template #obj>
+            <td *ngFor="let prop of this.columnConfig">{{row[prop]}}</td>
```

Comment:
Could you +1 to the `rank` property display instead of adding 1 in the model?

Reply:
Done